lib/panel.js

```diff
@@ -84,6 +93,14 @@ class Panel {
 		failedContainer.textContent = assertResult.currentExecution.failed;
 	}
 
+	_getCssClassForAssert(assert) {
+		if (assert.ok) {
+			return assert.skip ? 'skipped' : 'ok';
+		}
+
+		return assert.todo ? 'todo' : 'ko';
```

Comment:
Not certain about this comment. If I get it right, should be ok: 

The parser module that we're using has a flag 'ok' that will be active if the test passed. If the test is marked as todo, ok will be false and the todo flag will be true. Finally, failed tests will have ok as false, todo as false and therefore we get the 'ko' css class.

Unless I'm wrong, I think this code reflects these specs. Am I missing something?.